src/utilities/inputParsers.ts

```diff
@@ -0,0 +1,17 @@
+import { getInput } from '@actions/core';
+
+export enum AllowedMergeMethods {
+  MERGE = 'MERGE',
+  SQUASH = 'SQUASH',
+  REBASE = 'REBASE',
+}
+
+export const parseInputMergeMethod = (): AllowedMergeMethods => {
```

Comment:
What about the name `parseInputMergeMethodOrDefault`? 
Would be a bit more descriptive indicating that if the input is incorrectly formated it would ignore it and pass the default.

Also I would add a warn log indicating  that the input is being ignored because its malformed.